packages/component-catalog-spa/public/index.html

```diff
@@ -33,27 +33,14 @@
   </head>
 
   <body>
-
     <!-- #region NavBar -->
-    <script src="https://unpkg.com/@one-platform/opc-nav@0.0.3-prerelease/dist/opc-nav.js"
-      integrity="sha384-N/XXcy88sqNtDwdswFWIfew+ecxOGnny+j0CR7Bn/deliGg49PgoBVrK8IeHjrAI"
-      crossorigin="anonymous"></script>
-    <script src="https://unpkg.com/@one-platform/opc-menu-drawer@0.1.1-prerelease/dist/opc-menu-drawer.js"
-      integrity="sha384-lnyGND2Ba53ZPH1nFlneXIaszATrkbsa6EIqHY4TRhli1gGCD1HpCCLn9s6CeO2F"
-      crossorigin="anonymous"></script>
-    <script src="https://unpkg.com/@one-platform/opc-feedback@0.0.9-prerelease/dist/opc-feedback.js"
-      integrity="sha384-2rFZX1TmlNqGFKNQ7pcks2ddrToaIcGPGvN758FNeeUTg57uk8xJrZMiaygSTjhn"
-      crossorigin="anonymous"></script>
+    <script src="https://unpkg.com/@one-platform/opc-nav@latest/dist/opc-nav.js"></script>
```

Comment:
Is there a reason for going back to latest and removing source integrity?

Reply:
@akhilmhdh I suppose there would be more updates in SSI in the near future (example the search fix), hence change it each time would be time consuming. Once the SSI is stable it would be better to update with specific versions.

Reply:
But it could save 3  calls to just one